packages/updating-element/src/env.d.ts

```diff
@@ -0,0 +1,44 @@
+interface ShadyCSS {
+  nativeCss: boolean;
```

Comment:
is this required in updating element yet?

Reply:
It is now, yeah.